app/components/find/courses/about_schools_component/view.html.erb

```diff
@@ -30,7 +30,7 @@
       </h3>
 
       <p class="govuk-body">
-        The theoretical learning part of your course will be at the following location.
+        <%= "The theoretical learning part of your course will be at the following #{course.study_sites.many? ? 'locations' : 'location'}." %>
```

Comment:
Have we considered using translations to handle this? What happens if there are no study sites?

Reply:
Using a translation for this whole sentence? What's the benefit? This is the only place this sentence is used

If there are no study sites, we don't show any of this content, see: https://github.com/DFE-Digital/publish-teacher-training/pull/4346/files/8b2c06a7e295f556b67b282e995d9546d64b9775#diff-dc2b4f9394bd57a02c5edf89db885b96cc990e928d4185b5b863ae269f7e8954L27